kube/resources/volume-local.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
does this map onto the local fs somehow? or is there some song and dance we need to do to pull data off of it? obviously the former is preferred but if only the latter is possible we should at least document it.

Reply:
This is documented under `Copy Files To/From Volumes`.

After talking with Michel yesterday, it looks like we'll remove the local mount and mark it as not working on Kube, since it requires an NFS-like mount which we don't want to force people to support. 

If we improve the local mount to be a file destination like `source-file` is for sources, this problem is made easier since we can write to cloud storage and such.